disvae/losses.py

```diff
@@ -167,6 +169,100 @@ def __call__(self, data, recon_data, latent_dist, is_train, storer):
         return loss
 
 
+class FactorKLoss(BaseLoss):
+    """
+        Compute the Factor-VAE loss as per Algorithm 2 of [1]
+
+        Parameters
+        ----------
+        is_color : bool
+            Whether the image are in color.
+
+        discriminator : disvae.discriminator.Discriminator
+
+        optimizer_d : torch.optim.adam.Adam
```

Comment:
extremely minor note but this is is not restricted to Adam (i.e. `torch.optim`)

Reply:
I'll change this.